doc/source/tutorial/stats/outliers.md

```diff
@@ -0,0 +1,521 @@
+
+- trimming: removing outliers from the dataset, and
+- winsorization: replacing outliers with a more central value,
+
+using tools available in SciPy, with an emphasis on transitiong from use of legacy functions to the preferred approach.
+
+# Motivation
+In the past, SciPy has offered several "convenience functions" that combine trimming with computation of a statistic. Consider, for instance, `scipy.stats.trim_mean`.
+
+```{code-cell} ipython3
+import numpy as np
+from scipy import stats
+np.set_printoptions(linewidth=120)
+x = np.arange(20.)
+stats.trim_mean(x, proportiontocut=0.1)
+```
+
+Without carefully reading the documentation, it is not immediately obvious how the operation was performed. In this case, we would have gotten the same result whether `proportiontocut` refered to the proportion to cut from each tail or the total amount of data to remove.
+
+```{code-cell} ipython3
+# Remove 2/20 observations total before taking the `mean`, one in each tail
+np.mean(x[1:-1])
+```
+
+```{code-cell} ipython3
+# Remove 2/20 observations from *each* tail before taking the `mean`
+np.mean(x[2:-2])
+```
+
+Likewise, when the proportion would not result in an integral number of points being removed, how many points are removed? For example, in an array with 22 elements, is the number to remove rounded up or down?
+
+```{code-cell} ipython3
+x = np.arange(22.)
+# Remove `floor(0.1 * 22)` (2) observations from each tail
+np.mean(x[2:-2])
+```
+
+```{code-cell} ipython3
+# Remove `ceil(0.1 * 22)` (2) observations from each tail
+np.mean(x[3:-3])
+```
+
+What if instead of removing a proportion, we want to remove data below and above certain threshholds. It turns out that we would need to use `scipy.stats.tmean` or (the similarly-named, but curiously relegated to a separate namespace) `scipy.stats.mstats.trimmed_mean`, instead.
+
+```{code-cell} ipython3
+stats.tmean(x, limits=(2, 19), inclusive=(True, True))
+```
+
+```{code-cell} ipython3
+stats.mstats.trimmed_mean(x, limits=(2, 19), relative=False, inclusive=(True, True))
+```
+
+What if we want to remove data outside a "confidence interval" estimate. Do any of these functions - or perhaps `scipy.stats.trim` followed by `np.mean` - offer anything to facilitate that?
+
+No, not really.
+
+So at the time of writing, SciPy offers three or four ways of performing some operations related to trimming, but none are able to perform *all* operations we might reasonably expect them to. In the spirit of the Zen of Python, we suggest the new "one-- and preferably only one --obvious way to do it.": manually, in one or two lines of code, using more familiar, general-purpose functions. Whether calculating a trimmed or winsorized statistic, there will be three steps:
```

Comment:
"three or four ways of performing some operations related to trimming" - my mistake. There are three functions for the trimmed mean alone, and many more options for trimming.